src/sentry/snuba/metrics/naming_layer/mri.py

```diff
@@ -124,6 +124,8 @@ class TransactionMRI(Enum):
     ALL_USER = "e:transactions/user.all@none"
     USER_MISERY = "e:transactions/user_misery@ratio"
     TEAM_KEY_TRANSACTION = "e:transactions/team_key_transaction@none"
+    HTTP_ERROR_COUNT = "e:transactions/http_error_count@none"
```

Comment:
Based on the user guide, custom functions do not need an MRI. MRIs are for metric-like objects (including named expressions aka "derived metrics").

Reply:
I did this because exposing `http_error_count` as a derived metrics requires an MRI. I was going off of how failure rate was built. There's a separate count with its own MRI that is used to build the rate. https://github.com/getsentry/sentry/blob/master/src/sentry/snuba/metrics/fields/base.py#L1530